tests/test_save_load.py

```diff
@@ -183,17 +183,17 @@ def test_set_env(model_class):
     # create model
     model = model_class("MlpPolicy", env, policy_kwargs=dict(net_arch=[16]), **kwargs)
     # learn
-    model.learn(total_timesteps=300)
```

Comment:
A sidenote: Maybe we should invest bit of time to set all tests to default to "cpu" by default? All the cuda-initialization takes time and GPUs slow down small-network agents. Maybe leave "auto" for CnnPolicy tests.

Reply:
doing such thing would prevent us from finding bugs that are due to cuda only... (for instance a tensor that is not moved to the correct device)